### PR TITLE
Add cz-github-issues as an issue source for cz-component-dash in animations-ave.json

### PR DESCRIPTION
**Pull request by alancutter**
This change adds a new component dash to animations-ave.json showing web-animations-js polyfill issue statuses.

Preview:
image

This change builds upon https://github.com/shans/chromez/pull/76.
BUG=https://github.com/shans/chromez/issues/72
+@suzyh as author of cz-component-dash.

### CODE REVIEW

**Comment by suzyh** [comment]
lgtm

File: client/cz-component-dash.html [hunk 1]
```diff
@@ -112,25 +112,33 @@
...
         });
         registerSource('cz-config', 'updateSLO', updateSLO => {
           this.set('updateSLO', updateSLO);
         });
         registerSource('cz-config', 'issue-sources', issueSources => {
-          this.updateIssueSource(issueSources[this.key]);
+          var issueSource = issueSources[this.key];
+          registerSource(issueSource.tag, issueSource.query, issueList => {
+            this.set('issueList', issueList);
+          });
```
Comment (alancutter) [inline client/cz-component-dash.html:141]: Oops, thought I reverted the updateIssueSource() split in the previous patch.

File: client/cz-component-dash.html [hunk 2]
```diff
@@ -100,7 +100,7 @@
       is: "cz-component-dash",
 
       properties: {
-        users: {
+        userConfigs: {
```
Comment (alancutter) [inline client/cz-component-dash.html:103]: I can anticipate disapproval for this rename, it has a weaker line of reasoning than `searchQuery.query`.

File: client/cz-component-dash.html [hunk 3]
```diff
@@ -112,25 +112,33 @@
...
-        'updateComponent(issueList)',
+        'updateUsernames(userConfigs, issueList)',
+        'updateComponent(usernames, issueList)',
       ],
 
       attached: function() {
-        registerSource('cz-config', 'users', users => {
-          this.set('users', users.map(userData => userData['email']));
+        registerSource('cz-config', 'users', userConfigs => {
+          this.set('userConfigs', userConfigs);
```
Comment (shans) [inline client/cz-component-dash.html]: this.userConfigs = userConfigs

File: client/cz-component-dash.html (hunk 1 above)
Comment (shans) [reply client/cz-component-dash.html:141]: this.issueList = issueList :P

File: client/cz-component-dash.html [hunk 4]
```diff
@@ -141,21 +149,23 @@
...
+      updateComponent: function(usernames, issueList) {
         var team = new IssueList();
         var others = new IssueList();
         var unowned = new IssueList();
         if (issueList) {
           for (var issue of issueList) {
             if (issue.owner == null) {
               unowned.push(issue);
-            } else if (this.users.indexOf(issue.owner) === -1) {
+            } else if (!usernames.has(issue.owner)) {
```
Comment (shans) [inline client/cz-component-dash.html]: FYI, you can use .includes on a list directly now, so you don't need to Set-ify usernames (I don't think there's any other reason to do that here as it isn't edited anywhere).

File: client/cz-component-dash.html [hunk 5]
```diff
@@ -141,21 +149,23 @@
...
-      updateIssueSource: function({tag, query}) {
-        registerSource(tag, query, issueList => {
-          this.set('issueList', issueList);
-        });
+      updateUsernames: function(userConfigs, issueList) {
+        if (!issueList) {
+          return;
+        }
+        var usernames = userConfigs.map(userConfig => issueList.getUsername(userConfig));
+        this.set('usernames', new Set(usernames));
```
Comment (shans) [inline client/cz-component-dash.html]: this.usernames = ...

**Review by shans — APPROVED** [review]

**Comment by alancutter** [comment]
Changed usernames to an array, replaced some this.set() calls to assignment.

**Pull request merged**